src/feed-input.scss

```diff
@@ -27,6 +27,8 @@ $block: #{$fd-namespace}-feed-input;
 
   @include fd-disabled() {
     cursor: not-allowed;
+    pointer-events: initial;
+    opacity: initial;
```

Comment:
why?

Reply:
Without that the screen looks like this in NGX which is not seen in the style
<img width="1128" alt="Screenshot 2021-07-14 at 5 16 39 PM" src="https://user-images.githubusercontent.com/53534379/125616957-2b88278b-b8ba-4468-ab79-af981f94f5cf.png">

Reply:
will `initial` work in IE, have you checked? https://caniuse.com/?search=initial says no support for IE